tests/management/group/test_model.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2019 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""Test the group model."""
+from django.test import TestCase
+from tenant_schemas.utils import tenant_context
+from unittest.mock import Mock
+
+from management.models import Group, Role, Policy
+from tests.identity_request import IdentityRequest
+
+
+class GroupModelTests(IdentityRequest):
+    """Test the group model."""
+
+    def setUp(self):
+        """Set up the group model tests."""
+        super().setUp()
+
+        with tenant_context(self.tenant):
+            self.group = Group.objects.create(name='groupA')
+            self.roleA = Role.objects.create(name='roleA')
+            self.roleB = Role.objects.create(name='roleB')
+            self.policy = Policy(name='policyA', group=self.group)
+            self.policy.save()
+            self.policy.roles.add(self.roleA)
+            self.policy.save()
+            self.group.policies.add(self.policy)
```

Comment:
Wow, how great! This is our first model test! Just curious, could this line be removed?